src/components/common/SideMenu.js

```diff
@@ -127,21 +150,59 @@ class SideMenu extends React.Component {
   };
 
   onClickPauseOrPlay = () => {
-    const { dispatchSetIsPaused, isPaused } = this.props;
+    const {
+      dispatchSetIsPaused,
+      dispatchPostAction,
+      isPaused,
+      isMicroscopic,
+      electrons,
+      currentlyShowingKelvinScale,
+      showGrid,
+      showEmittedLines,
+      wavelengthDistribution,
+    } = this.props;
+    const appSettings = {
+      view: isMicroscopic ? MICROSCOPIC_STRING : MACROSCOPIC_STRING,
+      electrons,
+      scale: currentlyShowingKelvinScale ? KELVIN_STRING : CELSIUS_STRING,
+      grid: showGrid,
+      radiation: showEmittedLines,
+      wavelengthDistribution,
+    };
+    if (isPaused) {
+      dispatchPostAction({ verb: CLICKED_PLAY, data: { ...appSettings } });
+    } else {
+      dispatchPostAction({ verb: CLICKED_PAUSE, data: { ...appSettings } });
+    }
     dispatchSetIsPaused(!isPaused);
   };
 
   reset = () => {
-    const { dispatchResetSettings } = this.props;
+    const { dispatchResetSettings, dispatchPostAction } = this.props;
     dispatchResetSettings();
+    dispatchPostAction({ verb: CLICKED_RESET });
   };
 
   onToggleScaleUnit = () => {
-    const { dispatchSetScaleUnit, currentlyShowingKelvinScale } = this.props;
+    const {
+      dispatchSetScaleUnit,
+      currentlyShowingKelvinScale,
+      dispatchPostAction,
+      isPaused,
+    } = this.props;
+    const applicationState = isPaused ? PAUSED_STRING : PLAYING_STRING;
     if (currentlyShowingKelvinScale) {
       dispatchSetScaleUnit(SCALE_UNITS.CELSIUS);
+      dispatchPostAction({
+        verb: TOGGLED_TEMPERATURE_SCALE,
+        data: { newScale: CELSIUS_STRING, applicationState },
```

Comment:
Maybe I would go for `scale` instead of `newScale`.

Reply:
For consistency with the actions in the other apps I'll keep it as `newScale`. The reason I went with `newScale` is that in my experience actions can often be unclear (is this the scale we changed _from_ or that we changed _to_?). So I thought it would be nice to be explicit.